stock_picking_portal/models/res_config_settings.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2024 Cetmix OÜ
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    portal_visible_operation_ids = fields.Many2many(
+        comodel_name="stock.picking.type",
+        string="Portal Visible Operations",
+    )
+
+    def set_values(self):
+        super().set_values()
+        ICPSudo = self.env["ir.config_parameter"].sudo()
+        ICPSudo.set_param(
+            "stock_picking_portal.portal_visible_operation_ids",
+            ",".join(str(i) for i in self.portal_visible_operation_ids.ids),
+        )
+        return
+
+    @api.model
+    def get_values(self):
+        res = super().get_values()
+        ICPSudo = self.env["ir.config_parameter"].sudo()
+        portal_visible_operation_ids = ICPSudo.get_param(
+            "stock_picking_portal.portal_visible_operation_ids", default=False
+        )
+        if portal_visible_operation_ids:
+            res.update(
+                portal_visible_operation_ids=[
+                    int(r) for r in portal_visible_operation_ids.split(",")
+                ]
+            )
+        return res
```

Comment:
> Hello @ivs-cetmix, We put in ir.config;parameter the configurations that are global. If it is object level specific (could be partner, warehouse, route, picking type ...) then we put the configuration on the object. All this complex code can be replaced by just a boolean field on the picking type. You can still put a group on the field to restrict who can modify that.

Totally agree with you. This is why we put it in the configuration and not in the location. Because this is a global property of the portal. Not the property of the location.  
Eg when a user configures a portal this is much more convenient to do from a single view. Rather than to do several configurations in different places.
Same way as it's done for the POS starting Odoo 16

Reply:
hey @jbaudoux so what do we do next?
This PR is already used in several projects. Would be great to have it merged too.